baselines/dom.generated.d.ts

```diff
@@ -3519,40 +3519,6 @@ interface ConcatParams extends Algorithm {
     publicInfo?: Uint8Array;
 }
 
-/** Provides access to the browser's debugging console (e.g. the Web Console in Firefox). The specifics of how it works varies from browser to browser, but there is a de facto set of features that are typically provided. */
-interface Console {
-    memory: any;
```

Comment:
The following properties are now gone:
* memory
* exception
* [mark]timeline[End]

Do you know who this is likely to break? People with dependencies on old versions of IE? Somebody else?

Reply:
Seems that I have to keep `.memory` and `.exception` for now because they are in Edge and Firefox although not in spec. `.markTimeline` is only in Edge in non-functional state.